src/containers/alerts.jsx

```diff
@@ -14,7 +15,11 @@ const Alerts = ({
     onCloseAlert
 }) => (
     <AlertsComponent
-        alertsList={alertsList}
+        // only display standard and extension alerts here
+        alertsList={alertsList.filter(curAlert => (
+            curAlert.alertType === AlertTypes.STANDARD ||
+            curAlert.alertType === AlertTypes.EXTENSION
```

Comment:
`mapStateToProps` -- the problem (as I understand it) is that if this were in mapStateToProps, it would create a new object every time redux state changes, anywhere, and thus rerender on any redux state change. Whereas like this, the props don't change unless the actual content changes, and thus it only rerenders if the actual content changes.

Selector -- makes sense, will do.